pkg/util/defined_entity_util.go

```diff
@@ -46,11 +64,26 @@ func GetPVsFromRDE(rde *swaggerClient.DefinedEntity) ([]string, error) {
 	}
 
 	var pvInterfaces interface{}
-	if isCAPVCDEntityType(rde.EntityType) {
-		pvInterfaces = statusMap["persistentVolumes"]
-	} else if isNativeClusterEntityType(rde.EntityType) {
+	switch {
+	case isCAPVCDEntityType(rde.EntityType):
```

Comment:
I think this can also be exported and moved to vcdsdk as it is used in both CPI and CSI. Added to [VCDA-3682](https://jira.eng.vmware.com/browse/VCDA-3682)